packages/orchestrator-ui-components/src/components/WfoTable/WfoTableWithFilter/WfoTableWithFilter.tsx

```diff
@@ -200,15 +203,31 @@ export const WfoTableWithFilter = <T extends object>({
         return <WfoError />;
     }
 
+    const searchModalText = t.rich('searchModalText', {
+        br: () => <br />,
+        p: (chunks) => <p>{chunks}</p>,
+        ul: (chunks) => <ul>{chunks}</ul>,
+        li: (chunks) => <li>{chunks}</li>,
+        span: (chunks) => <span>{chunks}</span>,
+        b: (chunks) => <b>{chunks}</b>,
+    });
+
     return (
         <>
-            <EuiFlexGroup>
+            <EuiFlexGroup alignItems="center">
                 <EuiFlexItem>
                     <WfoSearchField
                         queryString={queryString}
                         onUpdateQueryString={onUpdateQueryString}
                     />
                 </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                    <EuiButtonIcon
```

Comment:
Is this according to desing? Don't we have something in our icons folder for this?

Reply:
It is not according to design, but we agreed on it